response_operations_ui/templates/collection-exercise.html

```diff
@@ -327,6 +337,8 @@ <h2 class="venus u-pt-m">Reference information for this collection exercise</h2>
               <td class="table--cell tbl-ce-value" name="survey-info">
                 {{survey.surveyRef}} {{survey.shortName}}
               </td>
+              <td class="table--cell tbl-ce-">
```

Comment:
Do we need the empty columns here?

Reply:
I added those in for the future if those columns ever needed to be updated, plus it visually looks better as some columns were uneven compared to the ones with pencils